### DISCUSSION TITLE
Make TagStudio read metadata on child directory from a root directory

### DISCUSSION THREAD

**Discussion by Endertainer007** [TagStudioDev/TagStudio#334] [Ideas]
So I've been playing around with the alpha version of TagStudio (v9.3.2) and have been pleased with it. It's everything I wanted to organize my library of reference images. But while testing the app, I found something.

So I made a test folder called `Child Test Library` where I'll store some of my reference images. There I added a few images and added Tags like Head, Expression, Frown, Smile, etc. Then I tried to put the `Child Test Library` into another folder called `Root Test Library` where I will also be adding a few of my reference images.

From this experiment, I learned that when I load the Root directory, the app loads all the images both from the Root and Child directory. But it doesn't load any tags I put in the Child directory. Instead, all the images from the Child directory don't have any tags I previously added. But when I load it back into the Child directory, all the tags are back.

<img width="975" alt="TagStudio_RsolFacxeT" src="https://github.com/user-attachments/assets/0b9274b2-716b-4464-b063-8954166b28fe">
<img width="975" alt="TagStudio_XWx44et3dn" src="https://github.com/user-attachments/assets/509a4d50-e45e-4bbd-8193-b81cf91078d2">

So in conclusion, I think it would be an important feature for the app to not only load any metadata like tags from the Root directory but also from any Child directory inside the Root directory.

But overall I'm amazed with the software and can't wait for the full release so I can finally start organizing my reference images.
